mscore/musescore.cpp

```diff
@@ -587,6 +589,86 @@ void MuseScore::populateNoteInputMenu()
             }
       }
 
+//---------------------------------------------------------
+//   populateFileOperations
+//---------------------------------------------------------
+
+void MuseScore::populateFileOperations()
+      {
+      fileTools->clear();
+
+      if (qApp->layoutDirection() == Qt::LayoutDirection::LeftToRight) {
+            for (auto s : _fileOperationEntries) {
+                  if (!*s)
```

Comment:
Dumb question - is it really (!*s) as opposed to (*s != "")?  Or does that conversion happen elsewhere?

Reply:
Most of this was just copying and pasting from existing code. I personally like `*s != ""` better if it works as expected

Reply:
Well, it segfaults when I tried that.  So, `!*s` checks the first character of s for `'\0'`

Reply:
Good point.  Like I said, dumb question :-)